test/unit/layout-grid/Cell.test.js

```diff
@@ -0,0 +1,55 @@
+import {assert} from 'chai';
+import {shallow} from 'enzyme';
+import React from 'react';
+
+import {Cell} from '../../../packages/layout-grid/index';
+
+suite('LayoutGridCell');
+
+test('classNames adds classes', () => {
+  const wrapper = shallow(<Cell className='test-class-name' />, {disableLifecycleMethods: true});
```

Comment:
why are you disabling the lifecycles in the test? is it faster?

Reply:
It is _slightly_ faster (mostly on larger test suites). I usually enable this for components that do not need to hook into lifecycle methods to just skip over them. The actual documentation (https://airbnb.io/enzyme/docs/api/shallow.html) doesn't say much about the impact on performance however. If you think this might be found confusing by future/other developers of this package I'll gladly remove it.

Reply:
We do use it sparingly throughout the repo. I don't think its necessary in this case. But its fine to keep it.